vaadin-ordered-layout-flow-parent/vaadin-ordered-layout-flow/src/main/java/com/vaadin/flow/component/orderedlayout/HorizontalLayout.java

```diff
@@ -55,6 +55,41 @@ public HorizontalLayout(Component... children) {
         add(children);
     }
 
+    /**
+     * Convenience constructor to create a layout with the children already
+     * inside it, and the default justifyContentMode.
+     *
+     * @param justifyContentMode
+     *            the default justifyContentMode
+     * @param children
+     *            the items to add to this layout
+     *
+     * @see #add(Component...)
+     * @see #setJustifyContentMode(JustifyContentMode)
+     */
+    public HorizontalLayout(FlexComponent.JustifyContentMode justifyContentMode,
+            Component... children) {
+        this(children);
+        setJustifyContentMode(justifyContentMode);
+    }
+
+    /**
+     * Convenience constructor to create a layout with the children already
```

Comment:
Nit (opinionated comment): would rephrase to `...with the children and specified vertical alignment`.